docs/Using Fleet/enroll-hosts.md

```diff
@@ -114,9 +115,28 @@ To generate an osquery installer for a team:
 3. Next, select **Add hosts** and copy the `fleetctl package` command for the platform (macOS, Windows, Linux) of the hosts you'd like to add to a team in Fleet.
 4. Run the copied `fleetctl package` command and [distribute your installer](#adding-multiple-hosts) to add your hosts to a team in Fleet.
 
+### Generating Windows installers using local WiX toolset v3 binaries on Windows
```

Comment:
Would consider shortening this header (since the details are spelled out on the next line)
```suggestion
### Generating Windows installers using local WiX toolset
```